packages/common/src/utils/dates/DateUtils.ts

```diff
@@ -20,4 +24,4 @@ export const DateUtils = {
   ): boolean => differenceInMonths(expiryDate, Date.now()) <= threshold,
 };
 
-export { isThisWeek, isToday };
+export { isThisWeek, isToday, isThisMonth, isAfter, isBefore, isEqual };
```

Comment:
oops. I think my intention was to export these as fields in the `DateUtils` object to be consistent. Otherwise it's a little difficult to know when something should be a part of the `DateUtils` object and when it should be exported by iteself which I guess by making the mistake myself first.. you've been caught by? 😁